suites/quincy/rgw/tier-1-extn_rgw.yaml

```diff
@@ -308,15 +308,14 @@ tests:
         script-name: test_Mbuckets_with_Nobjects.py
         config-file-name: test_bucket_check_fix.yaml
 
-  # - test:
-  #     name: Test functionality of user stat reset
-  #     desc: Test functionality of user stat reset
-  #     comments: Known issue BZ-2167475 targeted to 8.0
-  #     polarion-id: CEPH-83575439
-  #     module: sanity_rgw.py
-  #     config:
-  #       script-name: test_Mbuckets_with_Nobjects.py
-  #       config-file-name: test_user_stats_reset.yaml
+  - test:
```

Comment:
Hi @anrao19 ,

Here please consider a fact that few BZ's only fixed in later versions.
for ex: this BZ fix is given (8.0z2 onwards)

Reply:
made changes